pkg/license/identifier.go

```diff
@@ -80,3 +80,10 @@ func Identify(pkgPath, content string) (string, error) {
 
 	return "", fmt.Errorf("cannot identify license content")
 }
+
+var seemLicense = regexp.MustCompile(`(?i)licen[sc]e|copyright|copying`)
+
+// Seem determine whether the content of the file may be a license file
+func Seem(content string) bool {
+	return seemLicense.MatchString(content)
+}
```

Comment:
Can you reuse `possibleLicenseFileName` in file `golang.go`? it may not be a good idea to put `possibleLicenseFileName` in `identifier.go` but now let's unify the 2 places, you can move `possibleLicenseFileName` into `identifier.go` later.

Reply:
> 
> 
> Can you reuse `possibleLicenseFileName` in file `golang.go`? it may not be a good idea to put `possibleLicenseFileName` in `identifier.go` but now let's unify the 2 places, you can move `possibleLicenseFileName` into `identifier.go` later.

the purpose of `Seem` function was to simply determine whether one file may be a license base on the input content, not the file name. i remove it from `identifier.go` now.